### PR TITLE
docs: Update License Text

### PR DESCRIPTION
**Pull request by ThomasGalesloot**
# Pull Request Key Information

## Reason for change

Previously LDAR-Sim contained references to continued IM3S development which is not currently the case.

## What was changed

Removed mentions of IM3S in license text

## Intended Purpose

License clarity

## Level of version change required

N/A

## Testing Completed

N/A

## Target Issue

N/A

## Additional Information

N/A

### CODE REVIEW

**Comment by SEAJang** [comment]
I do not believe you have to maintain previous license messages when you move to MIT license, so removing all the messages should be fine. However, should there still be a mention of the Intelligent Methane Monitoring and Management System (IM3S) Group in the readme or something, as they were a significant contributor to LDAR-Sim development?

**Comment by SEAJang** [comment]
> > I do not believe you have to maintain previous license messages when you move to MIT license, so removing all the messages should be fine. However, should there still be a mention of the Intelligent Methane Monitoring and Management System (IM3S) Group in the readme or something, as they were a significant contributor to LDAR-Sim development?
> 
> Are you thinking a section gets added to the readme for acknowledgements and special thanks? I like that idea.

Yes.

**Review by SEAJang — APPROVED** [review]

**Comment by lgolston** [comment]
Somewhat off topics, but some <campaigns/methods, initialization/checks, initialization/campaigns, initialization/versioning, methods/init_func/repair_delay> of the source files do not have a license message. Should those be added?

File: README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ LDAR-Sim is free software: you can redistribute it and/or modify it under the te
 
 NOTE: This applies to all versions following Commit 69c27ec, Made on March 1st, 2021, Previous versions of LDAR-Sim were licensed under the GNU Affero General Public License. All redistributions or modifications made on LDAR-Sim versions created before Commit 69c27ec (March 1st, 2021) are required to be in compliance with version 3 of the GNU Affero General Public License.
 
+## Special Thanks and Acknowledgements
+
+ The LDAR-Sim development team would like to special thank and acknowledgement to the University of Calgary's Intelligent Methane Monitoring and Management System (IM3S) Group for their significant contributions to the development and growth of LDAR-Sim from 2018 to 2021.
```
Comment (lgolston) [inline README.md:42]:
```suggestion
 The LDAR-Sim development team would like to give a special thanks and acknowledgement to the University of Calgary's Intelligent Methane Monitoring and Management System (IM3S) Group for their significant contributions to the development and growth of LDAR-Sim from 2018 to 2021.
```

**Comment by SEAJang** [comment]
> Somewhat off topics, but some <campaigns/methods, initialization/checks, initialization/campaigns, initialization/versioning, methods/init_func/repair_delay> of the source files do not have a license message. Should those be added?

Added in the missing messages.

**Pull request merged**